Add Bank Hapoalim

dleshem opened the pull request:
Alexa rank: [6,413](https://www.alexa.com/siteinfo/bankhapoalim.co.il)

dleshem committed:
Add Bank Hapoalim

Carlgo11 committed:
Merge branch 'master' into poalim

dleshem committed:
Bank Hapoalim: fixed Twitter handle casing

Carlgo11 committed:
Merge branch 'master' into poalim

Carlgo11 commented on _data/banking.yml:
```diff
@@ -98,6 +98,13 @@ websites:
+    url: https://www.bankhapoalim.co.il/
+    img: poalim.png
+    twitter: PoaliMarkeTweet
+    facebook: BankHapoalimSocial
```
Can't find any page with that handle.

dleshem replied:
Twitter: https://twitter.com/PoaliMarkeTweet
Facebook: https://www.facebook.com/BankHapoalimSocial

Both work for me.

Carlgo11 replied:
The thing is, the Facebook link doesn’t work for me or the script. ![image](https://user-images.githubusercontent.com/3535780/69835386-7f87c580-1241-11ea-9859-7c194f37a334.jpg)

dleshem replied:
Perhaps they limited the page to Israeli users...

dleshem committed:
Merge branch 'master' into poalim

dleshem committed:
Bank Hapoalim: removed Facebook page

As requested by project owner (Facebook page is geo-limited to Israel, which breaks build)

Jawshy committed:
Update poalim icon
